recipes/jupyterlite-core/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "jupyterlite-core" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlite_core-{{ version }}.tar.gz
+  sha256: fea7ffe880f6efb16cd39855dd459ec3cf87b238de076dae518033926af1150a
+
+build:
```

Comment:
should be `noarch: python`: `importlib_metadata` is gonna be there anyway, due to deps _also_ wanting to be `noarch: python`, so there's no real harm in depending on it.

Reply:
https://github.com/conda-forge/doit-feedstock/blob/main/recipe/meta.yaml#L47